hooks/submitter_create.py

```diff
@@ -25,9 +25,17 @@ def __init__(self, *args, **kwargs):
 
         self.__app = self.parent
 
-        desktopclient_framework = self.load_framework(
-            "tk-framework-desktopclient_v0.x.x"
-        )
+        desktopclient_framework = None
+
+        try:
```

Comment:
I don't understand. Was is a breaking change? If not, why the bump to v1? It looks like the framework would still be compatible, no?

Reply:
Before 1.0.0 only works with Shotgun Create, you need 1.0.0 to have it working with ShotGrid Create

Reply:
Ok, so this is just a new feature. We should have bumped the minor version number instead if the APIs and protocols are the same.

Reply:
Yeah, this is what we should have done. Major versions mean something more in tk-frameworks than on apps. Do you want me to re-deploy?

Reply:
I'd appreciate it, thank you. Don't forget to update the release notes .